tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -794,9 +794,9 @@ class Converter {
         if (std::string(tensor->getName()).find("(Unnamed ITensor* ") ==
             std::string::npos) {
           LOG(WARNING) << "Quantization range was not found for "
-            << tensor->getName()
-            << ". This might be okay if TensorRT does not need the range"
-            << "(e.g. due to node fusion).";
+                       << tensor->getName() << ". "
```

Comment:
What will happen if after fusing there are still missing ranges? In this case IIUC the engine op will not have a null calibrator and will it crash? Please comment.

Reply:
If there are still missing ranges, Builder::buildCudaEngine will return nullptr. Which we catch in `ConvertGraphDefToEngine` and return `tensorflow::errors::Internal("Failed to build TensorRT engine")`.

Reply:
I see, thanks. Please document this behavior in both comment and the LOG message.